### PR TITLE
minor devui improvements

### PR DESCRIPTION
**Pull request by saddieeiddas**
This adds some very minor improvements to the devui.

- ensure maximize works
- tidy up close button

### CODE REVIEW

File: game/hud/src/components/DevUI/index.tsx [hunk 1]
```diff
@@ -340,8 +370,11 @@ class DevUI extends React.PureComponent<{}, ObjectMap<RootPage> | null> {
   public render() {
     if (!this.state) return null;
     const keys = Object.keys(this.state);
+    const anyMaximized = keys.reduce((p, k) => {
+      return (this.state[k].visible && this.state[k].maximized) ? true : p;
+    }, false);
     return (
-      <Container>
+      <Container style={{ zIndex: anyMaximized ? 10000 : 999 }}>
```
Comment (Ajackster) [inline game/hud/src/components/DevUI/index.tsx]: Doing this means that all DevUI's will cover the HUDWidgets (Settings, Game menu, etc.) which we don't want.

I would say just add a `DevUI` value after `Progression` to the HUDZOrder enum in game/hud/services/sessions/HUDZOrder.ts. Also, add a `MaximizedDevUI` property that goes before the GameMenu and Settings and use these values for the zIndex property.

File: game/hud/src/components/Watermark.tsx [hunk 2]
```diff
@@ -32,5 +32,5 @@ export interface WatermarkStyle {
 export default (props: {
   style?: Partial<WatermarkStyle>;
 }) => {
-  return <Watermark>Beta 1 - Do not stream or distribute.</Watermark>;
+  return <Watermark style={props.style ? props.style.watermark : {}}>Beta 1 - Do not stream or distribute.</Watermark>;
```
Comment (Ajackster) [inline game/hud/src/components/Watermark.tsx]: We should add a `pointer-events: none` to this guy so it doesn't block certain UI's (and there's no point it needs to capture mouse events). I can't click on the Maximize or X button on this widget.
cantclick

Comment (saddieeiddas) [reply game/hud/src/components/Watermark.tsx]: I have updated watermark to disable pointer events

File: game/hud/src/components/DevUI/index.tsx (hunk 1 above)
Comment (saddieeiddas) [reply game/hud/src/components/DevUI/index.tsx]: This is resolved, although I have not had a great deal of time to test everything is working nicely

**Comment by saddieeiddas** [comment]
@Ajackster I have updated the PR with requested changes.

**Comment by Ajackster** [comment]
Looks great @saddieeiddas thanks!

**Pull request merged**